tox.ini

```diff
@@ -11,10 +11,23 @@ deps =
   lint: pylint~=2.3.1
   mypy: mypy~=0.711
 
+setenv =
+  PYTHONPATH={toxinidir}/opentelemetry-api/src/
+  mypy: MYPYPATH={env:PYTHONPATH}
+
+changedir =
+  test: opentelemetry-api/tests
+
 
 commands =
-  py37-lint: pylint opentelemetry-api/src/opentelemetry/
+; Prefer putting everything in one pylint command to profit from duplication
+; warnings.
+  py37-lint: pylint opentelemetry-api/src/opentelemetry/ opentelemetry-api/tests/
   py37-mypy: mypy opentelemetry-api/src/opentelemetry/
+; For test code, we don't want to use the full mypy strictness, so we use its
+; default flags instead.
+  py37-mypy: mypy opentelemetry-api/tests/ --config-file=
+  test: python -m unittest discover
```

Comment:
Nit: python -m unittest is equivalent to python -m unittest discover

Reply:
Yeah, I saw that in the unittest docs, but went with "explicit is better than implicit"